### ISSUE TITLE
Attendee selection should allow selecting users without email

### ISSUE THREAD

**Issue by SebastianKrupinski** [nextcloud/calendar#6203]
### Is your feature request related to a problem? Please describe.

No

### Describe the solution you'd like

And organizer should be able to add an internal NC users without a email configured.

As the CalDav format requires a reference for an attendee (Email or other) some possible option for referencing an internal NC user might be...

DIR Parameter
```
ATTENDEE;CN=John Smith;DIR="nc://internal/user1(cn=3DBJim%20Dolittle)
```

**Comment by tcitworld** [comment]
Note : we should anticipate the format for federated attendees.

**Comment by SebastianKrupinski** [comment]
@tcitworld good point! the same parameter would be use to denote another NC instance
